content/docs/user-guide/configuration/minimal/index.en.md

```diff
@@ -1,7 +1,79 @@
 ---
 linkTitle: Minimal
-title: Minimal Configuration
+title: Minimal configuration
 weight: 10
 ---
 
-{{< language-switch 1 >}}
+At least three settings must be configured before you can use a new (or reset) OpenBikeSensor.
+
+* The left and right offsets as the largest width of the bike with rider (handlebar width, elbow, etc.)
+* API key
+* WLAN settings to upload your data
+
+## Enable configuration mode
+
+To activate the configuration mode, press and hold the button to turn on the device.
+
+There are two different modes of operation.
+
+**Access Point Mode**
+
+Without a configured WLAN in range, the OpenBikeSensor opens its own WLAN access point.
+* SSID is the MAC address of the device `OpenBikeSensor-xxxxxxxx`.
+* The default password is `12345678`.
+* The configuration page can be accessed via [http://172.20.0.1](http://172.20.0.1) in a web browser.
+
+**WiFi mode**
+
+If a previously configured WLAN is in range, the OpenBikeSensor will connect to it and show the IP address on its display. This is used to reach the configuration page in a device's browser in the same WLAN. 
+
+Data can only be uploaded from a WLAN with Internet. This can also be triggered directly from the OpenBikeSensor by pressing and holding down the button.
+
+## Login to the configuration menu
+Since version 0.6.x, an HTTPS connection is offered, and a PIN secures access to the web interface.
+
+* To open the configuration menu, click on **go to https**.
+* (Since version 0.6.x) When using HTTPS, a certificate warning is expected, which can be dismissed or silenced by importing the certificate into the browser. The certificate is presented for download in the dialog.
```

Comment:
Fix indentation.

```suggestion
  * (Since version 0.6.x) When using HTTPS, a certificate warning is expected, which can be dismissed or silenced by importing the certificate into the browser. The certificate is presented for download in the dialog.
```